src/lava/magma/runtime/message_infrastructure/include/ports.h

```diff
@@ -0,0 +1,208 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: BSD-3-Clause
+// See: https://spdx.org/licenses/
+
+#ifndef PORTS_H_
```

Comment:
Please merge to latest code in dev branch. The macro has been modified to INCLUDE_PORTS_H

Reply:
Updated and ran pylint

Reply:
@hexu33 CppLint gave an error that the macro should remain as PORTS_H_
https://github.com/lava-nc/lava/runs/7802504468?check_suite_focus=true